numalogic/connectors/druid.py

```diff
@@ -15,6 +15,67 @@
 _LOGGER = logging.getLogger(__name__)
 
 
+def make_filter_pairs(filter_keys, filter_values):
+    """
+
+    Args:
+        filter_keys: keys
+        filter_values: values.
+
+    Returns: a dict of key value pairs
+
+    """
+    filter_pairs = {}
+    for k, v in zip(filter_keys, filter_values):
+        filter_pairs[k] = v
+    return filter_pairs
+
+
+def build_params(aggregations, datasource, dimensions, filter_pairs, granularity, hours):
```

Comment:
Can we describe typing for variables please.
Everyone is not druid expert :)

Reply:
+1

Reply:
cool adding type hints